README.md

```diff
@@ -78,22 +79,26 @@ helmDefaults:
   # additional and global args passed to helm
   args:
     - "--set k=v"
```

Comment:
Would it worth noting that the args is empty by default, or is it too obvious to be documented?